src/devices/lacrosse_tx22uit.c

```diff
@@ -0,0 +1,135 @@
+/** @file
+    LaCrosse Technology View TX22U-IT temperature, humidity, wind speed/direction
+    and rain sensor
+
+    Copyright (C) 2020 Caz Yokoyama, caz at caztech dot com
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+/**
+LaCrosse Technology View TX22U-IT temperature, humidity, wind speed/direction
+and rain sensor
+
+Product pages:
+TBW
+
+Specifications:
+TODO: update for TX22U-IT
+- Wind Speed Range: 0 to 178 kmh
+- Degrees of Direction: 360 deg with 16 Cardinal Directions
+- Outdoor Temperature Range: -29 C to 60 C
+- Outdoor Humidity Range: 10 to 99 %RH
+- Update Interval: Every 31 Seconds
+
+http://nikseresht.com/blog/?p=99 tell protocol but my TX22U-IT is different protocol.
+*/
+
+#include <stdbool.h>
```

Comment:
plain `int` is prefered

Reply:
convert bool to int. and define FALSE and TRUE.

Reply:
Just use 0 and 1, we are dealing with bit-level stuff here anyway :)

Reply:
Done.